release/resources/module_list.json

```diff
@@ -311,6 +311,87 @@
         },
         {
             "name": "module-ballerinax-zendesk"
+        },
```

Comment:
Do we need to add these in the alphabetical order?

Reply:
The automated workflow will update the list alphabetically.